.github/workflows/npm-cd.yml

```diff
@@ -193,14 +192,14 @@ jobs:
         runs-on: ubuntu-latest
         steps:
             - name: Checkout
-              uses: actions/checkout@v3
+              uses: actions/checkout@v4
               with:
                   submodules: "true"
 
             - name: Install node
               uses: actions/setup-node@v3
               with:
-                  node-version: "16"
+                  node-version: "20"
```

Comment:
We are still using node 16 for testing. Do you want to update this as well?
If we have different versions in testing and releasing workflow, we release an untested version.

Reply:
We release with 20 since we get warning that 16 going to be deprecated.
We test with 16 so actually we test that the package working with 16, and node has backward compatibility so it should be enough.
For later stage we need also to find a way to replace the action of setup node 16 and manually install node 16 on all platform. 
But it is not urgent, and building this new CD process was already a big time consuming.
I'll leave this fix to post GA.